components/public-api/gitpod/experimental/v1/idp.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package gitpod.experimental.v1;
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/experimental/v1";
+
+service IDPService {
+  // GetIDToken produces a new IT token
+  rpc GetIDToken(GetIDTokenRequest) returns (GetIDTokenResponse) {};
```

Comment:
```suggestion
  rpc GetToken(GetTokenRequest) returns (GetTokenResponse) {};
```
It's misleading that it says ID, when the request/response objects don't refer to an `id` property

Reply:
The returned token is an IDToken in the OIDC sense. I added a link to the spec in the comment.